straxen/rucio.py

```diff
@@ -5,13 +5,16 @@
 import os
 import glob
 import hashlib
-import time
 from utilix import xent_collection
 import strax
+import warnings
+try:
+    import admix
+except (ImportError, RuntimeError):
+    print("Problem importing admix. You will not be able to use rucio tools.")
```

Comment:
Can we make this similar to https://github.com/XENONnT/straxen/pull/592? Otherwise we're getting an annoying message every time we do something outside of the container, e.g. for developers or on the DAQ.

Reply:
Ok this should be better now, thanks for the suggestion.